src/Api/IdentifiersExtractor.php

```diff
@@ -42,7 +42,7 @@ public function __construct(PropertyNameCollectionFactoryInterface $propertyName
         $this->resourceClassResolver = $resourceClassResolver;
 
         if (null === $this->resourceClassResolver) {
-            @trigger_error(sprintf('Not injecting %s in the CachedIdentifiersExtractor might introduce cache issues with object identifiers.', ResourceClassResolverInterface::class), E_USER_DEPRECATED);
+            @trigger_error(sprintf('Not injecting %s in the IdentifiersExtractor might introduce cache issues with object identifiers.', ResourceClassResolverInterface::class), E_USER_DEPRECATED);
```

Comment:
It could be changed to use `__CLASS__` instead? And remove "the".

Reply:
Is it a must have?